### DISCUSSION TITLE
How to use this extension?

### DISCUSSION THREAD

**Discussion by rkj** [joedevivo/vscode-circuitpython#60] [Q&A]
I just got a Adafruit Magtag. It works fine with Mu editor, but I much prefer VSCode, so would love this extension to work :)

I installed clean VSCode, Python extension and CircuitPython extension. I opened D:\ folder in the workspace, but any commands I do end with error like the one attached: 
![Screenshot](https://user-images.githubusercontent.com/4562/133908528-22ce376c-8fda-4b22-9c40-f9c843ce3d66.PNG)

Is there some additional initial configuration required?

**Comment by sberke** [comment]
been trying to get v0.1.19 to work to no avail.   Same errors when trying to use the commands.

![image](https://user-images.githubusercontent.com/8468870/194694938-d7e3d608-c24f-4244-b286-e1a809d551ea.png)

Starting using MSFT's [serial-monitor](https://marketplace.visualstudio.com/items?itemName=ms-vscode.vscode-serial-monitor) extension until circuitpython extension is worked out.
